about.html

```diff
@@ -12,53 +12,83 @@
 </head>
 
 <body>
+    <header>
+      <div id="navhead">
+          <img  id="headimg" src="./img/lambda-black.png" alt="black classic lambda logo">
+          <nav id="navi">
+              <ul id="navbar">
+                  <li class="navitems"><a href="./index.html">Home</a></li>
+                  <li class="navitems"><a href="./about.html">About</a></li>
+                  <li class="navitems">Products</li>
+                  <li class="navitems">Blog</li>
+                  <li class="navitems">Contact</li>
+              </ul>
+          </nav>
+      </div>
+      <div id="jumbopic">
+          <img id="jumboimg" src="./img/jumbo-about.png" alt="highway in dark with cars in super fast motion">
+      </div>
+    </header>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
+      <div class="bottomborders1" id="about"> 
+        <h2>About</h2>
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+      </div>
+
+      <!--seperating into divs so we can flex this section to align properly through page and make responsive-->
```

Comment:
In addition to divs to flexbox the content you can also still use semantic HTML as well.